### ISSUE TITLE
Identify orphaned blocks

### ISSUE THREAD

**Issue by Lvl4Sword** [CryptocurrencyExplorer/CryptocurrencyExplorer#7]
Connected to #19?

**Comment by dermoth** [comment]
not necessarily... if height has a multiple key then you can easily find orphans by searching for duplicate heights with no next blocks (exception for max height)...

What may take a few extra steps is finding the whole orphan links (ex where does it merge back to main? you can naively walk back until you get a single block for a given height but it's possible to have>2 forks on the same height...).

Else either link orphan blocks to an orphan_blocks table or add an orphan flag.

**Comment by Lvl4Sword** [comment]
This still needs done. I'll emphasize this in the new year.
